cmd/localCI/README.rst

```diff
@@ -0,0 +1,156 @@
+| LOCALCI_REPO_SLUG   | E.g foo/bar          | Repository owner / repository name                  |
++---------------------+----------------------+-----------------------------------------------------+
+
+
+Configuration file
+------------------
+The configuration file follows the syntax defined by the `Tom's Obvious, Minimal Language`_ and might contain next keys and sections:
+
+Keys
+~~~~
+
+- ``runTestsInParallel [bool] (optional)``: specify whether the tests can run in parallel. By default this value is ``false``. If **localCI** is running as a tool the value is ignored.
+
+Sections
+~~~~~~~~
+
+``[[Repo]]``
+............
+Contains information about a repository and can appear more than once.
+This section have next keys:
+
+- ``url [string] (mandatory)``: the URL to the repository.
+- ``pr [int] (optional)``: the pull request to test. If **localCI** is running as a service then only this pull request is monitored and tested.
+- ``refreshTime [string] (optional)``: specify the time to wait for checking if a pull request needs to be tested. This option is used only when
+  **localCI** runs as a service.  By default this value is 30s. For more information about formats and suffixes supported see `ParseDuration`_
+- ``token [string] (optional)``: the repository access token. If the control version repository imposes a rate limit,
+  this token is used to make the requests and post in the pull request the messages specified in ``postOnFailure``
+  and ``postOnSuccess``.
+- ``setup [[]string] (optional)``: the commands needed to setup the test environment.
+- ``run [[]string] (mandatory)``: the commands used to test the pull request once ``setup`` finished correctly.
+- ``teardown [[]string] (optional)``: the commands executed once ``run`` finished correctly.
+- ``onSuccess [[]string] (optional)``: the commands to be executed if ``setup``, ``run`` and ``teardown`` finished correctly.
+- ``onfailure [[]string] (optional)``: the commands to be executed if any of ``setup``, ``run`` or ``teardown`` fail.
+- ``postOnSuccess [string] (optional)``: message to be posted in the pull request if ``setup``, ``run`` and ``teardown`` finished correctly.
+  If this key is not present or is an empty string no mesage is posted.
+- ``postOnFailure [string] (optional)``: message to be posted in the pull request if any of ``setup``, ``run`` or ``teardown`` fail.
+  If this key is not present or is an empty string no mesage is posted.
+- ``useVM [bool] (optional)``: specify if **localCI** should use virtual machines to test the pull requests. By default this value is ``false``.
+- ``logDir [string] (optional)``: the directory where the logs are placed. By default this value is ``/var/log/localCI``
+
+and have next sub-sections
+
+- ``[Repo.language] (mandatory)``
+
+  The language of the project.
+
+  - ``language [string] (mandatory)``: the language of the project. To check the list of supported languages see `Supported languages`_
```

Comment:
Just a passing comment - in the future maybe we will need to support multiple languages per repo for mixed language repos.